lang/cs/Org.Apache.REEF.Driver/Bridge/Events/RunningTask.cs

```diff
@@ -60,12 +60,12 @@ public void Send(byte[] message)
 
         public void Suspend(byte[] message)
         {
-            throw new NotImplementedException();
+            _runningTaskClr2Java.Suspend(message);
         }
 
         public void Suspend()
```

Comment:
User can always call Suspend(null) if the user doesn't want to send message. Suspend() is a special case of Suspend(byte[]). If one can represent another, we don't need so many varieties.

I am OK to keep both if you think that is more explicit.